portal-api/src/v1/controllers/deal.controller.js

```diff
@@ -178,60 +173,54 @@ exports.delete = async (req, res) => {
  * @param {*} pagesize Size of each page - limits list results
  * @returns combined and formatted list of deals
  */
-const queryAllDeals = async (
-  filters = {},
-  sort = {},
-  start = 0,
-  pagesize = 0,
-) => {
+const queryAllDeals = async (filters = {}, sort = {}, start = 0, pagesize = 0) => {
   const startPage = computeSkipPosition(start, filters, sort);
 
   const collection = await db.getCollection('deals');
 
-  const results = await collection.aggregate([ // TODO SR-8
-    { $match: escapeOperators(filters) },
-    {
-      $project: {
-        _id: 1,
-        bankInternalRefName: '$bankInternalRefName',
-        status: '$status',
-        product: '$dealType',
-        submissionType: '$submissionType',
-        exporter: '$exporter.companyName',
-        // exporter in lowercase for sorting
-        lowerExporter: { $toLower: '$exporter.companyName' },
-        updatedAt: { $toDouble: '$updatedAt' },
+  const results = await collection
+    .aggregate([
+      { $match: escapeOperators(filters) }, // TODO SR-8 sanatise the filters
+      {
+        $project: {
+          _id: true, // TODO SR-8 Changed values in project to true or false to better represent the functionality
+          bankInternalRefName: '$bankInternalRefName',
+          status: '$status',
+          product: '$dealType',
+          submissionType: '$submissionType',
+          exporter: '$exporter.companyName',
+          // exporter in lowercase for sorting
+          lowerExporter: { $toLower: '$exporter.companyName' },
+          updatedAt: { $toDouble: '$updatedAt' },
+        },
       },
-    },
-    {
-      $sort: {
-        ...sort,
-        updatedAt: -1
+      {
+        $sort: {
+          ...sort, // TODO SR-8: Sort order only allows for $meta as an expression
+          updatedAt: -1, // TODO SR-8: Sort order -- no $eq expression needed
+        },
       },
-    },
-    {
-      $facet: {
-        count: [{ $count: 'total' }],
-        deals: [
-          { $skip: startPage },
-          ...(pagesize ? [{ $limit: pagesize }] : []),
-        ],
+      {
+        $facet: {
+          count: [{ $count: 'total' }],
+          deals: [
+            { $skip: startPage }, // TODO SR-8: Does not take an expression -- no $eq expression needed
+            ...(pagesize ? [{ $limit: pagesize }] : []), // TODO SR-8: Does not take an expression -- no $eq expression needed
+          ],
+        },
       },
-    },
-    { $unwind: '$count' },
-    {
-      $project: {
-        count: '$count.total',
-        deals: 1,
+      { $unwind: '$count' },
+      {
+        $project: {
+          count: '$count.total',
+          deals: true, // TODO SR-8 Changed values in project to true or false to better represent the functionality
```

Comment:
Remove todo once noted:
```suggestion
          deals: true,
```